src/lightning_app/cli/commands/connection.py

```diff
@@ -14,20 +15,24 @@
 from lightning_app.utilities.log import get_logfile
 from lightning_app.utilities.network import LightningClient
 
+_HOME = os.path.expanduser("~")
+_PPID = str(psutil.Process(os.getpid()).ppid())
+_LIGHTNING_CONNECTION = os.path.join(_HOME, ".lightning", "lightning_connection")
+_LIGHTNING_FOLDER = os.path.join(_LIGHTNING_CONNECTION, _PPID)
```

Comment:
Maybe should have a different name. when it says "lightning folder', I would think it is the .lightning

Reply:
Renamed to `_LIGHTNING_CONNECTION_FOLDER`